WordPress/src/main/res/layout/site_settings_format_dialog.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/text_title"
+        style="@style/Base.TextAppearance.AppCompat.Title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/image"
+        android:gravity="center_vertical"
+        android:padding="@dimen/margin_extra_large"
+        android:text="@string/site_settings_date_format_title" />
+
+    <RadioGroup
+        android:id="@+id/radio_group"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingLeft="@dimen/margin_extra_large"
+        android:paddingRight="@dimen/margin_extra_large" />
+
+    <EditText
+        android:id="@+id/edit_custom"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/margin_extra_extra_large"
```

Comment:
I really like to have both `layout_marginStart` and `layout_marginLeft` for RTL languages in my layouts. Do you think we should do that here? I think this is especially important when the left and right margins are different.

Of course we'd need `layout_marginEnd` as well. Also, it could be a good improvement for some other views in this layout.